onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3363,6 +3363,7 @@
       .Attr("N", "size of each output feature", AttributeProto::INT)
       .Attr("bits", "number of bits used for weight quantization (default 4)", AttributeProto::INT)
       .Attr("block_size", "number of groupsize used for weight quantization,(default 128). It needs to be a power of 2 and not smaller than 16.", AttributeProto::INT)
+      .Attr("accuracy_level", "minimum accuracy level, can be: 0(unset), 1(fp32), 2(fp16), 3(bf16), or 4(int8) (default unset)", AttributeProto::INT, static_cast<int64_t>(0))
```

Comment:
>    .Attr("accuracy_level", "minimum accuracy level, can be: 0(unset), 1(fp32), 2(fp16), 3(bf16), or 4(int8) (default unset)", AttributeProto::INT, static_cast<int64_t>( [](http://example.com/codeflow?start=2&length=169)

Could you please add more description, an example can be like:
minimum accuracy level of input A, can be: 0(unset), 1(fp32), 2(fp16), 3(bf16), or 4(int8) (default unset).
It is used to control how input A is quantized or downcast internally while doing computation, for example:
0 means input A will not be quantized or downcast while doing computation.
4 means input A can be quantized to int8 internally from type T1.

Reply:
added in 26d9bd7e6d4870c89d454931c189c9a93bf62488